retrieval/discovery/marathon/conversion.go

```diff
@@ -0,0 +1,78 @@
+package marathon
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/prometheus/client_golang/model"
```

Comment:
Throughout our code we always rename this via `clientmodel "github.com/prometheus/client_golang/model"`. 
I know that it seems pointless and there are plans to change that, but until then let's be consistent.